test/unit/models_test.rb

```diff
@@ -22,7 +22,7 @@ class ModelsTest < ActiveSupport::TestCase
       'LatestForumPostsPortlet' => %w[content_type], 'TableOfContentsPortlet' => %w[content_type], 'AuthenticationProvider::GitHub' => %w[branding_state account_type],
       'AuthenticationProvider::Keycloak' => %w[account_type], 'AuthenticationProvider::Auth0' => %w[account_type], 'AuthenticationProvider::Custom' => %w[account_type],
       'AuthenticationProvider::ServiceDiscoveryProvider' => %w[account_type], 'AuthenticationProvider::RedhatCustomerPortal' => %w[account_type],
-      'Account' => %w[credit_card_auth_code credit_card_authorize_net_payment_profile_token credit_card_partial_number],
+      'Account' => %w[credit_card_auth_code credit_card_authorize_net_payment_profile_token credit_card_partial_number service_preffix],
```

Comment:
This shouldn't be necessary because this test calls to `Account.columns` and here you are meant to say which of those active columns we don't need to test the string size for now.
https://github.com/3scale/porta/blob/04ae4112ad465694973b0a43d57f08edd78c8510/test/unit/models_test.rb#L35
But in here (https://github.com/3scale/porta/pull/1134/files#diff-1299db718a4681d9e44c3708ebbab542R9) you are removing `service_preffix` from what `Account.columns` returns so we don't need to discard it from here too because it is already discarded